src/domain/entities/Questionnaire/QuestionnaireRules.ts

```diff
@@ -45,11 +42,12 @@ export interface QuestionnaireRuleAction {
 }
 export interface QuestionnaireRule {
     id: Id;
-    condition: string; //condition is parsed with dataelementId e.g: #{dataElementId} == 'Yes'
+    condition: string;
```

Comment:
Here we need some kind of documentation to know that kind of things we can do in a condition.